docroot/themes/custom/uids_base/templates/layouts/onecol--background/layout--onecol--background.html.twig

```diff
@@ -21,6 +21,7 @@
     'banner--horizontal-left',
     'banner--large',
     'banner--vertical-bottom',
+    layout.template|clean_class,
```

Comment:
The way the styling is setup now, it doesn't make a difference visually, but both `layout--onecol--no--background` (from the preprocess) and `layout--onecol--background` (from the template name). The price of using a single template for both, and it's not causing any conflicts, but wanted to point it out as it seems off to me.

![image](https://user-images.githubusercontent.com/49074566/101032402-7eb5fd80-353f-11eb-95f5-5ddb7be4fc8e.png)

Reply:
@cory-skeers In the context of the banner, I don't think we need the preprocessed `layout--onecol--no--background` anymore. However, it is still being used in some of the SiteNow V2 scss files, so I left it in there for the time being.

Reply:
Okay! In that case I'd guess it's best to leave it in for now with both, and it can be a cleanup task post-V2. Thanks for the clarification.